pkgs/development/python-modules/pyverilog/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy3k
+, jinja2
+, ply
+, verilog
+, pytest
+, pytest-pythonpath
+}:
+
+buildPythonPackage rec {
+  pname = "pyverilog";
+  version = "1.3.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1a74k8r21swmfwvgv4c014y6nbcyl229fspxw89ygsgb0j83xnar";
+  };
+
+  disabled = !isPy3k;
+
+  propagatedBuildInputs = [
+    jinja2
+    ply
+    verilog
+  ];
+
+  checkInputs = [
+    pytest
```

Comment:
Please use pytestCheckHook.

Reply:
Thank you. Switched to that, but also kept the `pytest-pythonpath` `checkInput` to supress a warning.